app/controllers/users_controller.rb

```diff
@@ -142,6 +143,23 @@ def access_list_approvals
   def email
   end
 
+  # GET /facilities/:facility_id/users/:id/edit
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  # PUT /facilities/:facility_id/users/:id
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(params[:user])
```

Comment:
The code climate warning is legit: if I send a `username` param, it gets updated.